### PR TITLE
Capitalize some docstrings.

### PR DESCRIPTION
**Pull request by anntzer**
## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: lib/matplotlib/colors.py [hunk 1]
```diff
@@ -1356,8 +1356,7 @@ def rgb_to_hsv(arr):
 
 def hsv_to_rgb(hsv):
     """
-    convert hsv values in a numpy array to rgb values
-    all values assumed to be in range [0, 1]
```
Comment (anntzer) [inline lib/matplotlib/colors.py]: the same point is documented two lines below

File: lib/matplotlib/textpath.py [hunk 2]
```diff
@@ -68,11 +67,12 @@ def _get_char_id_ps(self, font, ccode):
...
     def glyph_to_path(self, font, currx=0.):
-        """
-        convert the ft2font glyph to vertices and codes.
-        """
+        """Convert the *font*'s current glyph to a (vertices, codes) pair."""
         verts, codes = font.get_path()
         if currx != 0.0:
+            cbook._warn_deprecated(
+                "3.1", name="currx", obj_type="argument",
+                alternative="Manually translate the path vertices.")
```
Comment (anntzer) [inline lib/matplotlib/textpath.py]: currx is not used in the codebase and that's really trivial for the caller to do it

File: lib/matplotlib/textpath.py [hunk 3]
```diff
@@ -287,9 +286,7 @@ def get_glyphs_mathtext(self, prop, s, glyph_map=None,
                 glyph_map_new, myrects)
 
     def get_texmanager(self):
-        """
-        return the :class:`matplotlib.texmanager.TexManager` instance
-        """
+        """Return a cached `~.texmanager.TexManager` instance."""
```
Comment (timhoffm) [inline lib/matplotlib/textpath.py]:
```suggestion
        """Return the cached `~.texmanager.TexManager` instance."""
```

There is just one instance.

File: lib/matplotlib/backend_tools.py [hunk 4]
```diff
@@ -853,7 +852,7 @@ def _cancel_action(self):
         return
 
     def _press(self, event):
-        """the _press mouse button in zoom to rect mode callback"""
+        """Callback for mouse button presses in zoom to rect mode."""
```
Comment (timhoffm) [inline lib/matplotlib/backend_tools.py]:
```suggestion
        """Callback for mouse button presses in zoom-to-rect mode."""
```
I would use hyphens for better readability, but not a native speaker, so not sure that's correct.

(occurs multiple times also in docstrings below)

Comment (anntzer) [reply lib/matplotlib/backend_tools.py]: done

File: lib/matplotlib/textpath.py (hunk 2 above)
Comment (QuLogic) [reply lib/matplotlib/textpath.py]: I really dislike slipping in deprecations in a commit that says it's about trivial changes to docstrings.

**Comment by timhoffm** [comment]
I agree with @QuLogic that we should separate documentation cleanup and functional changes. In particular, functional changes should be clear from the commit/PR message. It's not much work to do the deprecation in a separate PR.

File: lib/matplotlib/textpath.py (hunk 2 above)
Comment (anntzer) [reply lib/matplotlib/textpath.py]: Sorry about that, reverted.

**Comment by anntzer** [comment]
Apologies, reverted the deprecation.

File: lib/matplotlib/backend_tools.py (hunk 4 above)
Comment (QuLogic) [reply lib/matplotlib/backend_tools.py]: I don't see this?

Comment (anntzer) [reply lib/matplotlib/backend_tools.py]: Hum, sorry, forgot to push.  Done.

**Pull request merged**